### DISCUSSION TITLE
Change Region of Youtube Music

### DISCUSSION THREAD

**Discussion by maxrave-dev** [sigma67/ytmusicapi#377] [Ideas]
I build a backend using your API but my server is in US, and data return only of US. I see Youtube Music and Youtube also have change region feature, so can you add it to your API.
Thank you
Best Regard

**Comment by sigma67** [comment]
Hi, I think this would simply mean modifying the "gl" parameter in the headers. What do you hope to achieve by changing the region?

**Reply by maxrave-dev** [reply]
Can you tell me more about "gl"? I search in Google but no information

**Reply by sigma67** [reply]
It's just the equivalent of the region setting in the headers. What do you hope to achieve by changing the region?

**Reply by maxrave-dev** [reply]
I use ytmusicapi to create a Flask app and deploy to Heroku, but heroku server is in Ireland, and Home data return only Ireland content in client. So i want to custom region and data return which in client region
